common/datavalues2/src/columns/group_hash.rs

```diff
@@ -36,7 +36,7 @@ impl Series {
         })
     }
 
-    pub fn serialize(column: &ColumnRef, vec: &mut Vec<Vec<u8>>) -> Result<()> {
+    pub fn serialize(column: &ColumnRef, vec: &mut [Vec<u8>]) -> Result<()> {
```

Comment:
for `clippy::ptr_arg`, may need to check https://github.com/rust-lang/rust-clippy/issues/8334

Reply:
We can try to merge it first and then modify it if there is indeed a performance problem.

Reply:
I also found that!!!